packages/ERTP/src/paymentLedger.js

```diff
@@ -458,56 +379,115 @@ export const vivifyPaymentLedger = (
     issuerBaggage,
     name,
     assetKind,
-    getBrand,
+    brand,
+    PurseIKit,
     harden({
       depositInternal,
       withdrawInternal,
     }),
   );
 
-  const issuer = vivifySingleton(issuerBaggage, `${name} issuer`, {
-    getBrand: () => brand,
-    getAllegedName: () => name,
-    getAssetKind: () => assetKind,
-    getDisplayInfo: () => displayInfo,
+  /** @type {Issuer} */
+  const issuer = vivifyFarInstance(issuerBaggage, `${name} issuer`, IssuerI, {
+    getBrand() {
+      return brand;
+    },
+    getAllegedName() {
```

Comment:
Do we need to update the Wiki re [Arrow Function style](https://github.com/Agoric/agoric-sdk/wiki/Arrow-Function-Style)?

Reply:
Yes, good point. I'll do that and leave this conversation unresolved until then.

Reply:
Done.